spec.html

```diff
@@ -4819,6 +4819,20 @@ <h1>The ClassElementDefinition Record Specification Type</h1>
               The initializers of the field or accessor, if any.
             </td>
           </tr>
+          <tr>
+            <td>
+              [[ExtraInitializers]]
```

Comment:
This PR doesn't seem to include the algorithm steps that add entries to this list, so it will always be empty.

Reply:
`[[ExtraInitializers]]` are passed into `ApplyDecoratorsToElementDefinition`, which then creates a context object which adds the initializers there. The logic there is a bit funky, I think I can clean it up at some point so that decorators always use `[[ExtraInitializers]]` and we don't have the 
`_staticMethodExtraInitializers_` and `_instanceMethodExtraInitializers_`, those were from a bit of an earlier draft before there was much distinction between extra initializers